src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,7 +18,14 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+* Support Point In Time Restore
```

Comment:
please use past tense for "Support". thanks

Reply:
Fixed